src/common/stores/store-names.ts

```diff
@@ -16,16 +16,8 @@ export enum StoreNames {
     PathSnippetStore,
     UnifiedScanResultStore,
```

Comment:
Does "unified" here refer to the "unified" (i.e. Android) client? Should it therefore be removed?